course/src/main/java/org/openedx/course/presentation/unit/video/VideoFullScreenFragment.kt

```diff
@@ -187,6 +189,7 @@ class VideoFullScreenFragment : Fragment(R.layout.fragment_video_full_screen) {
         super.onPause()
         exoPlayer?.removeListener(exoPlayerListener)
         exoPlayer?.pause()
+        activity?.window?.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
```

Comment:
You added the flag in onCreate and clear it in onPause. Maybe it would be better to clear it in onDestroy?

Reply:
As only one activity is used, it is better to move the `keepScreenOn` at the view level IMO.

eg. `playerView.keepScreenOn = true`

PS: +1 with @dixidroid's comment to utilizing the `onResume` and `onPause`.